src/types.ts

```diff
@@ -208,6 +208,9 @@ interface QuoteBase {
     protocolFee: BigNumber;
     minimumProtocolFee: BigNumber;
     allowanceTarget?: string;
+    // Our calculated price impact or undefined if we were unable to
+    // to calculate any price impact
+    estimatedPriceImpactPercentage: BigNumber | undefined;
```

Comment:
NIT(just opinion): I know we already do this, but using `undefined` to represent no value in API responses is kind of a pet peeve o mine. The key will disappear when the JSON is serialized and it can be confusing to the consumer of the API, I personally prefer using `null` values.

Reply:
oh yeah I agree that it disappearing is actually extremely annoying